source/_integrations/flume.markdown

```diff
@@ -21,7 +21,9 @@ Flume monitors the real-time status of your home water meter. Allowing the end-u
 
 {% include integrations/config_flow.md %}
 
-You can find your Client ID and Client Secret under "API Access" on the [settings page](https://portal.flumewater.com/#settings). 
+You can find your Client ID and Client Secret under "API Access" on the [settings page](https://portal.flumewater.com/#settings).
+
+*NOTE: If you purchased your Flume after August 23rd 2021, Flume requires a paid subscription to access the API, even for personal use.*
```

Comment:
```suggestion
Flume API integration is available to Flume Insight users only. 
```

Reply:
I want to keep the idea that it's a subscription in because these documents do influence purchasing decisions and it's a relevant piece of information.  Wording it as "Flume Insight users only" implies that one simply needs to make a Flume Insight account without conveying the implications of the account.

My original wording was perhaps a tad antagonistic, but the first part about the date is quite relevant too since those users are grandfathered in based on purchase date.

Reply:
@bdraco what do you think?

Reply:
I don't think we want to document Flume's business logic / subscription dates / grandfathering since it may change. A broad statement that you might need to pay for api access seems ok though.  Maybe:

`The Flume API access is available to Flume Insight users only, which may require a paid subscription.`

Reply:
Sounds good to me; updated with that text.